tests/testthat/test-stan_expected_obs.R

```diff
@@ -0,0 +1,25 @@
+skip_on_cran()
+skip_on_os("windows")
+skip_on_os("mac")
+skip_on_local()
+
+enw_stan_to_r(
+  c("expected_obs.stan", "hazard.stan"),
+  system.file("stan", "functions", package = "epinowcast")
+)
+# note these tests require enw_stan_to_r() to be run first
+
+test_that("expected_obs() aggregates probabilities correctly", {
+  tar_obs <- 0
+  lh <- c(0.1, 0.2, 0.3, 0.4)
```

Comment:
Will also need to edit the test case in light of the log-prob vs. prob mistake

Reply:
Worth thinking about what dev steps could have taken to spot this sooner (though given this is still in draft its pretty good). For me I didn't run the test case - just sight read which I think would have helped me catch it